microshift.py

```diff
@@ -66,8 +67,14 @@ def generate_kickstart(rhel_number: str, uname_m: str) -> None:
     with open('pull_secret.json', 'r') as f_in:
         file_contents = f_in.read()
 
-    with open('/root/.ssh/id_rsa.pub', 'r') as ssh_in:
-        ssh_contents = ssh_in.read()
+    pub_files = glob.glob('/root/.ssh/*.pub')
+    if pub_files:
+        with open(pub_files[0], 'r') as ssh_in:
+            ssh_contents = ssh_in.read()
```

Comment:
We already have some kind of key detection logic somewhere else. Can we unify the two places?

Reply:
Yes, I can make a function for both `clusterDeployer.py`  and `microshift.py` to call